syntax = "proto3";

package users;

// The greeting service definition.
service User {
	// Unary
	rpc UserAuth (userRequest) returns (userReply);

}

// The request message containing the user's name.
message userRequest {
  string client_key = 1;
}

// The response message containing the greetings.
message userReply {
  int32 id = 1;
  string user_name = 2;
  string is_superuser = 3;
  string is_staff = 4;
  repeated string permission_list= 5;
}
